src/chapters/vstools/studio.rst

```diff
@@ -0,0 +1,22 @@
+If It Ain't Broke, Add a Breakpoint!
+====================================
+
+On your machine, open up your fork of the ``HelloASPDotNETDemo`` `repository <https://github.com/LaunchCodeEducation/HelloASPDotNETDemo>`_ in Visual Studio.
+The purpose of this studio is to talk about your current debugging strategies and how to make the most of the debugger tools discussed in this chapter.
+
+Before we start practicing with debugging tools, go over with the group one error you encountered when working on your own version of ``HelloASPDotNET``.
```

Comment:
how about adding some suggestions of types of errors they may have seen to get the ball rolling?

Reply:
Added some things that could have gone for students with the application!